Loosen constraints for Laravel 5.6

dwightwatson opened the pull request:
I know it was only released a couple of hours ago but this extends the support for Laravel 5.5 to include 5.6 as well. This allows the current version of the package to work with both versions.

It also upgrades Testbench for Laravel 5.6 so that the package is tested against it, as well as upgrading PHPUnit as that's the current version used by Laravel.

dwightwatson commented:
Ah whoops, just realised that Laravel 5.6 bumps up the minimum PHP requirement to 7.1.3.

We could leave the tests to run against 5.5 only (so that they will run on PHP 7.0) or consider a new major version for Laravel 5.6. Do you have any preference on that?

davejamesmiller commented:
I think a new major version supporting Laravel 5. 6 / PHP 7. 1 only would
be easiest to maintain in the long-run.

Thanks
Dave

On 8 Feb 2018 02:24, "Dwight Watson" <notifications@github.com> wrote:

Ah whoops, just realised that Laravel 5.6 bumps up the minimum PHP
requirement to 7.1.3.

We could leave the tests to run against 5.5 only (so that they will run on
PHP 7.0) or consider a new major version for Laravel 5.6. Do you have any
preference on that?

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/davejamesmiller/laravel-breadcrumbs/pull/165#issuecomment-363981665>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/AAOcSJGiMZmBJVhv_af-osbZH3zjUUtSks5tSlr5gaJpZM4R9sZN>
.

dwightwatson commented:
Cool, I've updated this to reflect your preference - supporting only Laravel 5.6 in a new major release. Also updated Travis to test against 7.1 and 7.2 because that seems reasonable.

freekmurze commented:
Looking forward to the new release!

davejamesmiller commented:
Thanks, I'll get this released as soon as I can over the weekend.

On 8 Feb 2018 15:16, "Dwight Watson" <notifications@github.com> wrote:

Cool, I've updated this to reflect your preference - supporting only
Laravel 5.6 in a new major release. Also updated Travis to test against 7.1
and 7.2 because that seems reasonable.

—
You are receiving this because you commented.

Reply to this email directly, view it on GitHub
<https://github.com/davejamesmiller/laravel-breadcrumbs/pull/165#issuecomment-364143111>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/AAOcSIQU4iH1iCdKhhOetQEpfPNRY1GHks5tSw_egaJpZM4R9sZN>
.

freekmurze commented:
For my packages I've just loosened the constraints with creating a new major version. There were literally zero breaking changes.

Should you opt to go this route as well, take a look at this PR: https://github.com/davejamesmiller/laravel-breadcrumbs/pull/166

davejamesmiller commented:
@freekmurze Thanks, but I'll use the opportunity to drop PHP 7.0 support. :-)

freekmurze commented:
Great idea!

davejamesmiller commented:
Released in 5.0.0.
